pkg/channel/distributed/controller/kafkachannel/controller.go

```diff
@@ -20,22 +20,21 @@ import (
 	"context"
 	"sync"
 
+	"knative.dev/eventing-kafka/pkg/channel/distributed/common/config"
+
```

Comment:
nit, can you move this down?

Reply:
Sure, as long as the go formatting bot doesn't mind :)